learning_resources/etl/loaders.py

```diff
@@ -268,37 +268,45 @@ def load_course(  # noqa: C901
             )
             return None
 
-        if deduplicated_course_id:
-            # intentionally not updating if the course doesn't exist
-            (
-                learning_resource,
-                created,
-            ) = LearningResource.objects.select_for_update().get_or_create(
-                platform=platform,
-                readable_id=deduplicated_course_id,
-                resource_type=LearningResourceType.course.name,
-                defaults=resource_data,
-            )
-
-            if readable_id != deduplicated_course_id:
-                duplicate_resource = LearningResource.objects.filter(
-                    platform=platform, readable_id=readable_id
-                ).first()
-                if duplicate_resource:
-                    duplicate_resource.published = False
-                    duplicate_resource.save()
-                    resource_unpublished_actions(duplicate_resource)
```

Comment:
Seemed more complicated than necessary, so simplified it.  Relevant unit test still passes.